ats/pkb/parents.py

```diff
@@ -0,0 +1,14 @@
+def parent(list):
+    all_children = set(list.children)
+    pool = all_children.copy()
+    while pool:
+        child = pool.pop()
+        all_children.add(child)
+        pool.update(child.children)
+        if "stmtLst" in child.__str__():
+            if len(child.children) > 0:
+                print(child.children)
+                return True
+            else:
+                print(child.children)
+                return False
```

Comment:
To sprawdza jedynie czy istnieje stmtLst, ktory ma dzieci, A ISTNIEJE bo taki jest wymog specyfikacji